ts/nni_manager/core/sqlDatabase.ts

```diff
@@ -20,7 +20,15 @@ import { TrialJobDetail } from '../common/trainingService';
 
 
 const createTables: string = `
-create table TrialJobEvent (timestamp integer, trialJobId text, event text, data text, logPath text, sequenceId integer, message text);
+create table TrialJobEvent (
+    timestamp integer,
+    trialJobId text,
+    event text,
+    data text,
+    logPath text,
+    sequenceId integer,
+    message text,
+    environmentId text);
```

Comment:
you changed the table which means the experiment created in previous version nni cannot be resumed by 3.0, right?

Reply:
yes

Reply:
then you should give this notice in a proper place

Reply:
I think it's enough in release note?